src/app/modules/ix-forms/components/ix-select/ix-select.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="full-width container has-tooltip">
 
-  <div *ngIf="label" class="label-container">
+  <div *ngIf="label || tooltip" class="label-container">
```

Comment:
how it would be rendered if the tooltip is present but the label is missing?

Reply:
Network -> Global Configuration
![image](https://user-images.githubusercontent.com/22980553/150725369-2d4a3686-2280-4648-bf42-ab34266df2ab.png)

There is no label, only tooltip.